docs/javascript/v2/how--to-guides/set-up-video-conferencing/render-video/overview.mdx

```diff
@@ -35,6 +35,13 @@ hmsActions.detachVideo(videoTrack.id, videoElement);
 
 ## When to attach/detach videos
 
+Starting from this [release](https://www.100ms.live/docs/javascript/v2/changelog/release-notes#2023-03-14), sdk handles the subsequent attach/detach automatically.
+- attach needs to be called only the first time video element is available.
+- detach needs to be called only when the element is removed from dom.
```

Comment:
detach needs to be called only when the element is removed from the dom while the track is still available. For the majority of cases, calling detach won't be needed.